### PR TITLE
2019-01-19-spidernet-romania.md

### PR DESCRIPTION
**Pull request by donealex231993**

### CODE REVIEW

**Comment by donealex231993** [comment]
Adăugarea serverului meu pe site-ul azeroth

File: _posts/2019-01-19-spidernet-romania.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Spidernet Romania | WoW Private Server
+website: https://www.spidernetromania.net
+author: Alex
+image:https://www.spidernetromania.net/application/images/misc/banner.jpg (suggested dimensions width: 468px  height: 60px )
```
Comment (FrancescoBorzi) [inline _posts/2019-01-19-spidernet-romania.md]: only put the image link, remove "  (suggested dimensions width: 468px  height: 60px )"

File: _posts/2019-01-19-spidernet-romania.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Spidernet Romania | WoW Private Server
+website: https://www.spidernetromania.net
+author: Alex
+image:https://www.spidernetromania.net/application/images/misc/banner.jpg (suggested dimensions width: 468px  height: 60px )
+tags: wotlk pve pvp Blizzlike 1x all and 5x xp 
+published: true
+---
+
+Testimonial text: I tested a repack for the first time and saw that almost 100% of the important quests and quests go. And I compiled an Azeroth core hardly because I had some errors but I managed to get it off
```
Comment (FrancescoBorzi) [inline _posts/2019-01-19-spidernet-romania.md]: you can remove "Testimonial text:"

File: _posts/2019-01-19-spidernet-romania.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: Spidernet Romania | WoW Private Server
+website: https://www.spidernetromania.net
+author: Alex
+image:https://www.spidernetromania.net/application/images/misc/banner.jpg
```
Comment (FrancescoBorzi) [inline _posts/2019-01-19-spidernet-romania.md]: please add a space after `:`

**Pull request closed without merging**